grafana-pyroscope.yaml

```diff
@@ -0,0 +1,96 @@
+package:
+  name: grafana-pyroscope
+  version: 1.13.1
+  epoch: 0
+  description: Continuous Profiling Platform. Debug performance issues down to a single line of code
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - cairo-dev
+      - go
+      - node-gyp
+      - nodejs-20
+      - pango-dev
+      - python3
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 4797950b45e3fbf1f44e704819cd1dfd81b78f0c
+      repository: https://github.com/grafana/pyroscope
+      tag: v${{package.version}}
+
+  - runs: |
+      # https://github.com/grafana/pyroscope/blob/3da96b8e449de267d4663e14207b8b272f9edc6d/.github/workflows/release.yml#L48C14-L48C33
+      mkdir -p pyroscope/scripts/webpack
+      mkdir -p pyroscope/public/app
+      mkdir -p pyroscope/public/templates
+      cp yarn.lock package.json tsconfig.json ./pyroscope
+      cd pyroscope
+      yarn --frozen-lockfile
+      cp -r ../scripts/webpack ./scripts/
+      cp -r ../public/app ./public/
+      cp -r ../public/templates ./public/
+      yarn build
+      cp -r ./public/build ../public/build
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/grafana/pyroscope/pkg/util/build.Branch=main
+        -X github.com/grafana/pyroscope/pkg/util/build.Version=${{package.version}}
+        -X github.com/grafana/pyroscope/pkg/util/build.Revision=$(git rev-parse HEAD)
+        -X github.com/grafana/pyroscope/pkg/util/build.BuildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
+      output: pyroscope
+      packages: ./cmd/pyroscope
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/grafana/pyroscope/pkg/util/build.Branch=main
+        -X github.com/grafana/pyroscope/pkg/util/build.Version=${{package.version}}
+        -X github.com/grafana/pyroscope/pkg/util/build.Revision=$(git rev-parse HEAD)
+        -X github.com/grafana/pyroscope/pkg/util/build.BuildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
+      output: profilecli
+      packages: ./cmd/profilecli
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}"/etc/pyroscope
+      install -Dm755 ./cmd/pyroscope/pyroscope.yaml "${{targets.contextdir}}"/etc/pyroscope/config.yaml
+
+update:
+  enabled: true
+  github:
+    identifier: grafana/pyroscope
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - name: "Verify help command output"
+      runs: |
+        profilecli -h
+        pyroscope -h
+    - name: "Start and verify operator"
+      uses: test/daemon-check-output
+      with:
+        start: |
+          pyroscope
+        timeout: 30
+        expected_output: |
+          Pyroscope started
+          server listening on addresses
+        post: |
+          # check for the ready endpoint
+          curl -s http://localhost:4040/ready
```

Comment:
there's also `/metrics` endpoint which we can query and grep some metrics (it has one buildinfo metric in here) 

would be awesome if we can do something more with profilectl as well if possible else we can cover the same at images.

Reply:
done brother, thanks!